src/resources/views/user/home.blade.php

```diff
@@ -113,36 +86,18 @@
                     <h2 class="my-2 text-white text-center">Microblog Newsfeed</h2>
                 </div>
                 <ul class="list-group">
+                    @foreach ($posts as $post)
                     <li class="list-group-item" style="background-color: #FFFFF0">
                         <h5 class="card-title">User Name</h5>
-                        <p class="card-text">Sample1 post of the User</p>
```

Comment:
you can a create a component of a post

Reply:
I have resolved this comment, sir. Thank you! 🙇